lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -38,6 +38,11 @@ def carve_hostname_on_disk(hostname)
           true
         end
 
+        def attach(server_id)
+          service.attach_disk(id, server_id)
+          true
```

Comment:
I agree with you.
But please see other methods I pasted below. All methods return TRUE.
Just I follow the same style...

I think we need another refactoring for this model.

```
  class SakuraCloud
      class Disk < Fog::Model
...
        def save
          requires :name, :plan, :source_archive
          data = service.create_disk(@attributes[:name], @attributes[:plan], @attributes[:source_archive]).body["Disk"]
          merge_attributes(data)
          true
        end

        def configure(sshkey_id)
          requires :id
          service.configure_disk(@attributes[:id], sshkey_id )
          true
        end

        def carve_hostname_on_disk(hostname)
          requires :id
          service.carve_hostname_on_disk(@attributes[:id], hostname )
          true
        end

        def associate_ip(ipaddress, networkmasklen, defaultroute)
          subnet ={
            :ipaddress => ipaddress,
            :networkmasklen => networkmasklen,
            :defaultroute => defaultroute
          }
          requires :id
          service.associate_ip_to_disk(@attributes[:id], subnet )
          true
        end
```